src/context/Providers.tsx

```diff
@@ -1,11 +1,14 @@
 import React from "react";
+import LeafyGreenProvider from "@leafygreen-ui/leafygreen-provider";
```

Comment:
This allows us to change all our base font sizes for the leafygreen typography components.